www/widgets/dukpy.js

```diff
@@ -16,6 +16,7 @@ addEventListener("message", (e) => {
             val = eval?.(e.data.body);
         } catch (e) {
             console.log('Script crashed');
+            console.log(e);
```

Comment:
Yeah this is fine but we can probably use like `console.trace` or `console.error`, right? Not really worth optimizing I guess.